src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Export/ResourceToNdjsonBytesSerializer.cs

```diff
@@ -18,21 +16,12 @@ namespace Microsoft.Health.Fhir.Core.Features.Operations.Export
     /// </summary>
     public class ResourceToNdjsonBytesSerializer : IResourceToByteArraySerializer
     {
-        private readonly FhirJsonSerializer _fhirJsonSerializer;
-
-        public ResourceToNdjsonBytesSerializer(FhirJsonSerializer fhirJsonSerializer)
-        {
-            EnsureArg.IsNotNull(fhirJsonSerializer, nameof(fhirJsonSerializer));
-
-            _fhirJsonSerializer = fhirJsonSerializer;
-        }
-
         /// <inheritdoc />
         public byte[] Serialize(ResourceElement resourceElement)
         {
             EnsureArg.IsNotNull(resourceElement, nameof(resourceElement));
 
-            string resourceData = _fhirJsonSerializer.SerializeToString(resourceElement.ToPoco<Resource>());
+            string resourceData = resourceElement.Instance.ToJson();
```

Comment:
shall we add FhirJsonSerializationSettings in "ToJson()"

Reply:
To keep consistent with before, the default setting is used here.